src/pybind/mgr/prometheus/module.py

```diff
@@ -759,6 +759,18 @@ def _setup_static_metrics(self) -> Dict[str, Metric]:
             'healthcheck status by type (0=inactive, 1=active)',
             HEALTHCHECK_DETAIL
         )
+        metrics['pool_objects_repaired'] = Metric(
```

Comment:
Since you would need to push anyway, please add a blank line before this line and before `metrics['daemon_health_metrics'] = Metric(` for consistency with definitions above.